server/http.go

```diff
@@ -238,8 +238,28 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 		return
 	}
 
-	post.Message = "Meeting has ended."
+	start := time.Unix(0, post.CreateAt*int64(time.Millisecond))
+	length := int(((model.GetMillis() - post.CreateAt) / 1000) / 60)
+	startText := start.Format("Mon Jan 2 15:04:05 -0700 MST 2006")
+	topic, ok := post.Props["meeting_topic"].(string)
+	if !ok {
+		topic = "Zoom Meeting"
+	}
+
+	slackAttachment := model.SlackAttachment{
+		Fallback: fmt.Sprintf("Meeting %s has ended: started at %s, lenght: %d minute(s).", post.Props["meeting_id"], startText, length),
+		Title:    topic,
+		Text: fmt.Sprintf(
+			"Personal Meeting ID (PMI) : %d\n\n##### Meeting Summary\n\nDate: %s\n\nMeeting Length: %d minute(s)",
```

Comment:
@jespino Something feels wrong in this formatting. Take a look to the screenshot.
![Screenshot from 2020-05-29 18-33-22](https://user-images.githubusercontent.com/1933730/83283294-45d94c80-a1db-11ea-95f3-ed4bca110591.png)

Reply:
Yes, I have to investigate that, will be fixed soon :)